.github/workflows/openssl-version-bump.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump OpenSSL version
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run daily
+    - cron: "0 0 * * *"
```

Comment:
This is going to produce PRs that basically always have conflicts with the boring ones, won't it?

Reply:
Or make it all one job 😬

Reply:
We could! could maybe even parametrize the job?

Reply:
Parameterize would have the same 2x PR problem, right?